packages/apps/tools/src/pages/transactions/cross-chain-transfer-finisher/index.tsx

```diff
@@ -302,23 +304,41 @@ const CrossChainTransferFinisher: FC = () => {
     "sigs": ${pollResults.tx?.receiverGuard.keys.map((key) => `"${key}"`)}
   }`;
 
+  const linkToExplorer = `${getExplorerLink(
+    requestKey,
+    networkData.networkId,
+    networksData,
+  )}`;
+
+  const renderLinkToExplorer = (
+    <p>
+      <span className={notificationKeyStyle}>
+        {t('Target Chain Request key: ')}
+      </span>
+      <Link
+        className={notificationLinkStyle}
+        href={linkToExplorer}
+        target={'_blank'}
+        key={requestKey}
+      >
+        {requestKey}
+      </Link>
+    </p>
+  );
+
   const renderNotification =
     txError.toString() === '' && receiverRequestKey ? (
       <FormStatusNotification
         status="successful"
         title={t('Notification title success')}
         body={t('XChain transfer has been successfully finalized!')}
       >
-        <p
-          className={noticationKeyStyle}
-        >{`Request key: ${receiverRequestKey}`}</p>
```

Comment:
So this is now also `"Target Chain Request key:"` (instead of `"Request key:"`), that is okay?

Reply:
it's already cahnged